MeetupPlatformApi/Features/Meetups/GetMeetups/Filter/FilterSettings/MeetupsFilterSettings.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.GetMeetups.Filter.FilterSettings;
+
+using System.ComponentModel.DataAnnotations;
+
+public class MeetupsFilterSettings
+{
+    [Required]
+    [Range(1,100000)]
+    public int PageNumber { get; set; }
+    
+    [Required]
+    [Range(1, 100, ErrorMessage = "too much data")]
```

Comment:
There is no need to specify an error message. On most UIs user can only choose from a strict set of predefined page sizes, so this message will never be exposed to the end user.